proto/internalapi/central/compliance_operator.proto

```diff
@@ -113,6 +113,18 @@ message ComplianceResponse {
     }
 }
 
+// Represents the severity of the rule
+enum ComplianceOperatorRuleSeverity {
+    UNSET_RULE_SEVERITY = 0;
+    UNKNOWN_RULE_SEVERITY = 1;
+    INFO_RULE_SEVERITY = 2;
+    LOW_RULE_SEVERITY = 3;
+    MEDIUM_RULE_SEVERITY = 4;
+    HIGH_RULE_SEVERITY = 5;
+}
+
+// ComplianceOperatorCheckResultV2 is a message from Sensor (to Central) representing a compliance check result.
+// Next tag: 16.
 message ComplianceOperatorCheckResultV2 {
```

Comment:
Moved the enum out for re-use.  Rest of the changes in this object are spacing related to that.